Exercise_sequential_basic/sample_array_job.bash

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#SBATCH --partition=reservation                                                                                                                                                                                    
+#SBATCH --reservation=fall_training_cpu_2022
+#SBATCH --job-name=test_array
+#SBATCH --time=00:05:00
+#SBATCH --nodes=1
+#SBATCH --cpus-per-task=1
+#SBATCH --output=%A_%a.out
+#SBATCH --error=%A_%a.err
+#SBATCH --array=1-4%4 #run a 4-job array, 4 jobs will launched at the same time.
+
+# Use a regular partition, such as express, when not in a training                                                                                                                                                                  
+# session                                                                                                                                                                                                                   
+##SBATCH --partition=express 
```

Comment:
I think this line will be ignored by slurm because there's a space between the `#SBATCH` directives. so might be good to just point it out during the training, or in the comments.

Reply:
@marianalevineu , I'm actually going to revise this as per @dhml's suggestion.

Reply:
and done @dhml & @marianalevineu